src/components/ItemInfo/ItemInfo.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+type Props = {
+  item: string,
+};
+
+export const ItemInfo: React.FC<Props> = ({ item }) => {
```

Comment:
In this situation, such a division into components is a little inappropriate, since the removed component takes up much more space than dividing the tag into a straight line. It is worth separating complex components or components that are lexically different from each other, but if we have a list with simple tags, it is worth keeping everything together.